jerry-core/vm/opcodes.c

```diff
@@ -75,7 +75,7 @@ vm_var_decl (vm_frame_ctx_t *frame_ctx_p, /**< interpreter context */
  * @return ecma value
  *         Returned value must be freed with ecma_free_value
  */
-ecma_value_t
+inline ecma_value_t
```

Comment:
`JERRY_ATTR_ALWAYS_INLINE` is missing

Reply:
Hm, we could just remove this function.

Reply:
Well, since it's only a "jumper" function, may worth to remove it.